docs/support_table.md

```diff
@@ -51,8 +51,8 @@ $\gdef\VERT{|}$
 |\]|$]$||
 |{|${a}$|`{a}`|
 |}|${a}$|`{a}`|
-|\\{|$\{$||
-|\\}|$\}$||
+|\\{|$\\{$||
+|\\}|$\\}$||
```

Comment:
It seems doesn't work
![image](https://github.com/KaTeX/KaTeX/assets/28651011/15ff9fde-3b93-4b2a-97ec-82e6d0573b55)

Reply:
Indeed (see the deploy preview below, or build the docs locally). Too bad the fix isn't so easy! It might be the Markdown plugin needs some fixing.

As a fallback, we could use `\lbrace` and `\rbrace`, but that wouldn't be ideal because it's not actually testing the specified function.